<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfBM_PlatformStatus</class>
 <widget class="QWidget" name="WkfBM_PlatformStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>621</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Platform Status</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="ut::qt::FixedColumnTableView" name="tableView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="currentColumnName"/>
         </item>
         <item>
          <widget class="QToolButton" name="addColumnButton">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Add Aux Data Column</string>
           </property>
           <property name="text">
            <string>Add Aux Data</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="typeLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="ruleSets"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="controlsLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="restoreButton">
           <property name="text">
            <string>Restore Defaults</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ut::qt::FixedColumnTableView</class>
   <extends>QWidget</extends>
   <header>UtQtFixedColumnTableView.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>addColumnButton</tabstop>
  <tabstop>restoreButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
